### DISCUSSION TITLE
python3-12

### DISCUSSION THREAD

**Discussion by terribleJim** [bareboat-necessities/lysmarine_gen#475] [General]
I installed home assistant using the install guide for BBN. It is giving me a warning (in home assistant) that python 3.11.2 is being deprecated and to install python 3.12. In terminal I did do a apt update and apt upgrade, which didn't upgrade python apparently. I don't want to break anything in case something depends on the older python version. This is latest BBN on rpi4.

**Comment by mgrouch** [comment]
Ignore the warning and don't attempt upgrade
